dev-python/jupyter_core/jupyter_core-4.3.0.ebuild

```diff
@@ -50,6 +53,5 @@ python_test() {
 }
 
 python_install_all() {
-	use doc && HTML_DOCS=( docs/_build/html/. )
 	distutils-r1_python_install_all
 }
```

Comment:
chuck this whole phase function, just calling the default is what happens anyway